library/core/src/sync/atomic.rs

```diff
@@ -3266,7 +3283,8 @@ unsafe fn atomic_umax<T: Copy>(dst: *mut T, val: T, order: Ordering) -> T {
 
 /// returns the min value (unsigned comparison)
 #[inline]
-#[cfg(target_has_atomic = "8")]
+#[cfg_attr(not(bootstrap), cfg(target_has_atomic))]
+#[cfg_attr(bootstrap, cfg(target_has_atomic = "8"))]
```

Comment:
This one makes sense though. It will only be codegened for sizes that are actually used and because it is private, this doesn't expose the ability to codegen an incorrect atomic size to the user.